passphrase/passphrase_test.go

```diff
@@ -165,5 +165,5 @@ func TestGetPassphrase(t *testing.T) {
 	stdin := bufio.NewReader(&in)
 	passphrase, err := GetPassphrase(stdin)
 	require.NoError(t, err)
-	require.Equal(t, string(passphrase), "passphrase\n")
+	require.Equal(t, "passphrase", string(passphrase))
```

Comment:
@riyazdf this was necessary due to the change in passphrase.go just above. I'm second guessing this change now though. I think it's probably just Windows being Windows, but I'm wondering if there's any reason it would break on Linux. I think this is _only_ used for logging in to notary, not any key passphrases.

Reply:
Oh, and I swapped the order around because it was wrong :P `require.Equal` wants expected then actual. The test failure message was confusing me for a minute.

Reply:
`GetPassphrase` is also used in `requestPassphrase`, which means that it's also used for key passphrases